index.js

```diff
@@ -4,9 +4,11 @@ module.exports = {
   },
   "extends": "eslint:recommended",
   "rules": {
-    "prefer-const": "warn",
-    "no-var": "warn",
-    "prefer-template": "warn",
+    "prefer-const": "error",
+    "no-var": "error",
+    "prefer-template": "error",
+    "max-params": ["error", 3],
```

Comment:
yea, we differ on opinions here.  It's not that the function is doing too many things, it's that we now require passing of many diff pieces of data to call our db wrappers for example.  I think we can still jsdoc the expected keys, it's just a PITA to keep appending args to a function and then inserting `null` or `undefined` in cases where the 2nd arg is not required